### ISSUE TITLE
Can't build docker image

### ISSUE THREAD

**Issue by pgiraud** [dalibo/temboard#211]
I tried to build the docker image using `make build` in the docker directory. However it fails because of issue with the installation of `postgresql-client`.

Here are the commands to reproduce:
```
$ docker container run --interactive --tty --rm python:2.7-slim bash
# apt-get update && apt-get install -y postgresql-client
```

I get the following trace:
```
update-alternatives: using /usr/share/postgresql/9.4/man/man1/psql.1.gz to provide /usr/share/man/man1/psql.1.gz (psql.1.gz) in auto mode
update-alternatives: error: error creating symbolic link `/usr/share/man/man1/psql.1.gz.dpkg-tmp': No such file or directory
dpkg: error processing package postgresql-client-9.4 (--configure):
 subprocess installed post-installation script returned error exit status 2
dpkg: dependency problems prevent configuration of postgresql-client:
 postgresql-client depends on postgresql-client-9.4; however:
  Package postgresql-client-9.4 is not configured yet.

dpkg: error processing package postgresql-client (--configure):
 dependency problems - leaving unconfigured
Processing triggers for libc-bin (2.19-18+deb8u10) ...
Processing triggers for systemd (215-17+deb8u7) ...
Errors were encountered while processing:
 postgresql-client-9.4
 postgresql-client
E: Sub-process /usr/bin/dpkg returned an error code (1)
```

I could not find any relevant information elsewhere.

**Comment by pgiraud** [comment]
Here's an ugly fix. https://github.com/dalibo/temboard/commit/ff98d6740ae11345658508b02052294d6cffd448

**Comment by bersace** [comment]
cf. https://github.com/debuerreotype/debuerreotype/issues/10

**Comment by bersace** [comment]
@pgiraud je pense que ton hack est le bon. `update-alternatives` a été corrigé dans Debian sid.

**Comment by bubthegreat** [comment]
This fix works for later ones as well (Fixed 9.6 install)
